doc/devdocs/modules/launcher/plugins/timedate.md

```diff
@@ -0,0 +1,110 @@
+# 'Time and Date' plugin
+The 'Time and Date' plugin shows the date and time in different formats. For the date and time formats the plugin uses the culture setting in Windows, if the format is not commonly defined. The user can search for the system date/time or a custom date/time. The value of each result can be copied to clipboard.
+
+![Image of Window Walker plugin](/doc/images/launcher/plugins/timedate.png)
+
+![Image of Window Walker plugin](/doc/images/launcher/plugins/timedate2.png)
+
+
+**Query examples:**
+- Format: `time`
+- Date/time: `10:30 AM`
+- Format and date/time: `Week number::10/10/2022`
+
+
+## Formats
+
+The following formats are currently available:
+
+| Format | Example (Based on default settings) | As result | As input |
+|--------------|-----------|------------|------------|
+| Time | 5:10 PM | x | x |
+| Date | 3/5/2022 | x | x |
+| Now | 3/5/2022 5:10 PM | x | x |
+| Time UTC | 4:10 PM | x | x |
+| Now UTC | 3/5/2022 4:10 PM | x | x |
+| Unix Timestamp | 1646496622 | x | x |
+| Day (Week day) | Saturday | x | |
+| Day of the week | 6 | x | |
+| Day of the month | 5 | x | |
+| Day of the year | 64 | x | |
+| Week of the month | 1 | x | |
+| Week of the year (Calendar week, Week number) | 10 | x | |
+| Month | March | x | |
+| Month of the year | 3 | x | |
+| Year | 2022 | x | |
+| Windows file time (Current date and time as Int64 number) | 637820976123938199 | x | x |
+| Universal time format: YYYY-MM-DD hh:mm:ss| 2022-03-05 16:20:12Z | x | x |
+| ISO 8601 | 2022-03-05T17:23:04 | x | x |
+| ISO 8601 UTC | 2022-03-05T16:23:04 | x | x |
+| ISO 8601 with time zone | 2022-03-05T17:23:04+01:00 | x | x |
+| ISO 8601 UTC with time zone | 2022-03-05T16:23:04Z | x | x |
+| RFC1123 | Sat, 05 Mar 2022 16:23:04 GMT | x | x |
+
+### Remarks
+
+- The following formats requires a prefix in the query:
+   - Unix Timestamp: `u`
+   - Windows file time: `ft`
+- On invalid number inputs we shaw a warning that tells the user which prefixes are allowed/required.
+
+## Adding new formats
+- To add a new formats you have to add them in the method `GetAvailableResults()` of the [`ResultHelper`](/src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.TimeDate/Components/ResultHelper.cs) class.
+	- Please add the new formats in the second range. The first one is reserved for the following three main formats (Time, Date, Now).
+- After adding the new formats you have to update the Unit Tests!
```

Comment:
- move into technic section after scrore section; replace with a not about the moved part here
- add not "New methods for date/time conversation/formatting can be added in the 'xyz' class"